### PR TITLE
DMP-1002 logon without 2FA

### PR DESCRIPTION
**Pull request by trev-hmcts**
### Change description ###
Added step to handle logon without 2FA & methods in classes NavigationShared
Get environment variables for userid & password - changes in class ReadProperties
Removed (most) commented out code from changed classes 
Added some code from DMP-847 that was neccessary to compile

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+					.assertThat().statusCode(204)
+					.extract().response();
+		return response;
+    }
+    
+
+@Test
+	public void test() {
+    	JsonApi jsonApi = new JsonApi();
+//		System.out.println("post: courthouses");
```
[the remark this answers is not in the document]
Comment (trev-hmcts) [reply src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java:245]: Those are all for testing / debuging the code & most of those that rely on changing data are commented out but would want to update & use if code changed
They don't get executed when automation is running

File: src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+        ResponseSpecification loglevel = new ResponseSpecBuilder().log(loggingLevel).build();
+        return loglevel;
+    }
+    
+    public String authenticate() {
+    	log.info("authentication");
+    	response  = 
+    		given()
+    			.spec(requestLogLevel(ReadProperties.authRequestLogLevel))
+    			.header("Content-Type","application/x-www-form-urlencoded")
```
Comment (karen-hedges) [inline src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java:58]: Should these strings all be constants as they are repeated throughout?

File: src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+//    			+ "}");
+//		System.out.println("get2: courthouses");
+//		api.getApi("courthouses");
+//		System.out.println("delete: courthouses");
+//		deleteApi("courthouses/2", "{\n"
+//    			+ "  \"courthouse_name\": \"Leeds\",\n"
+//    			+ "  \"code\": 100\n"
+//    			+ "}");
+//		System.out.println("get2: courthouses");
+//    	getApi("courthouses");
```
Comment (karen-hedges) [inline src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java:265]: Commented out code should be removed as git manages any history of changes. If its a temporary change then you should have a description explaining why the code is commented out.

File: src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java (hunk 2 above)
Comment (trev-hmcts) [reply src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java:58]: I am refactoring this class as part of DMP-847 & will do that there

File: src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java (hunk 3 above)
Comment (trev-hmcts) [reply src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java:269]: I am refactoring this class as part of DMP-847 & will do that there

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**